build.gradle

```diff
@@ -531,7 +531,7 @@ subprojects { Project subproject ->
                 configFile = rootProject.file('config/checkstyle/checkstyle.xml')
 
                 // Per submodule
-                maxErrors = 5
+                maxErrors = 0
                 maxWarnings = 10
```

Comment:
I left warning be since TODO statements were marked as warnings and those probably shouldn't cause a broken build.